web/packages/teleport/src/config.ts

```diff
@@ -483,6 +486,11 @@ const cfg = {
     return generatePath(cfg.routes.users, { clusterId });
   },
 
+  getBotsRoute() {
```

Comment:
Should this have a `clusterId` arg like the others?

Reply:
poke